docs/PrismaMigrationTools.md

```diff
@@ -0,0 +1,58 @@
+# Prisma Migration Tools
+
+## `npm prisma:seed`
+
+### What it does
+
+This command seeds the database, which means populating the database with initial data it requires to run.
+It can be dummy data, basic user accounts, a default language, etc.
+
+### Why to use it
+
+While developing the application, we need some data to be present in order for something to appear on the front end to interact with. Running this command populates the database with a basic set of data for development purposes.
+
+For more detailed information about [prisma seed](https://www.prisma.io/docs/guides/database/seed-database), click the link.
+
+## `npm prisma:reset`
+
+### What it does
+
+This command drops the database if possible or performs a soft reset if the database cannot be deleted.
+If the database is dropped, it creates a new database with the same name and runs all migrations and seed scripts.
+If `npm prisma:reset` runs without any errors, then the `npm prisma:seed` function is auto-invoked, meaning you do not need to run the `npm prisma:seed` command again.
+
+### Why to use it
+
+If you need to undo manual changes or if there are migration history conflicts, you should run this command.
+
+You use this command to also restore the data to its original state if you sufficiently modified the data in the front end.
+
+For more detailed information about [prisma reset](https://www.prisma.io/docs/concepts/components/prisma-migrate), click the link.
+
+## `npm prisma:migrate`
+
+### What it does
+
+This command enables you to keep your database schema in sync with the prisma schema.
+
+### Why to use it
+
+After you make a change to the prisma schema, you should run this function to update the database schema.
+Each time you run this command, it generates an SQL file that is added to the `migrations` folder.
```

Comment:
actually it only creates a new SQL file if there's changes to the prisma schema that haven't already been turned into SQL. and you only need to name the newly created migration if SQL file(s) are created. if no new migration is created, this command simply runs the existing SQL files against the database to apply the existing migrations.